resqpy/multiprocessing/wrappers/grid_surface_mp.py

```diff
@@ -158,19 +172,31 @@ def find_faces_to_represent_surface_regular_wrapper(
         _, p = surface.triangles_and_points()
         pset = PointSet(model, points_array = p, crs_uuid = grid.crs.uuid, title = surface.title)
         surface = Surface(model, crs_uuid = grid.crs.uuid, title = pset.title)
-        surface.set_from_point_set(pset,
-                                   convexity_parameter = 2.0,
-                                   reorient = True,
-                                   extend_with_flange = extend_fault_representation,
-                                   flange_radial_distance = flange_radius,
-                                   make_clockwise = False)
+        flange_bool = surface.set_from_point_set(pset,
+                                                 convexity_parameter = 2.0,
+                                                 reorient = True,
+                                                 extend_with_flange = extend_fault_representation,
+                                                 flange_radial_distance = flange_radius,
+                                                 make_clockwise = False)
         del pset
         extended = True
         retriangulated = True
     if not bu.matching_uuids(surface.uuid, surface_uuid):
         surface.write_hdf5()
         surface.create_xml()
         surface_uuid = surface.uuid
+    if flange_bool is not None:
+        flange_bool_property = Property.from_array(parent_model = model,
+                                                   cached_array = flange_bool,
+                                                   source_info = 'flange bool array',
+                                                   keyword = 'flange bool property',
```

Comment:
let's drop 'property' from the title here; 'flange bool' or simply 'flange' is better I feel